tutorials/run-symfony-on-appengine-flexible/index.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Run Symfony on Google App Engine Flexible Environment
+description: Learn how to deploy a Symfony app to Google App Engine flexible environment.
+author: bshaffer
+tags: App Engine, Symfony, PHP
+date_published: 2017-03-15
+---
+## Symfony
+
+> [Symfony][symfony] is a set of PHP Components, a Web Application framework, a
+> Philosophy, and a Community — all working together in harmony.
+>
+> – symfony.com
+
+You can check out [PHP on Google Cloud Platform][php-gcp] to get an
+overview of PHP itself and learn ways to run PHP apps on Google Cloud
+Platform.
+
+## Prerequisites
+
+1. Create a project in the [Google Cloud Platform Console](https://console.cloud.google.com/).
```

Comment:
When I go to this link, it's not immediately clear to me how to create a new project.